packages/react-utils/src/createRequiredContext.ts

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react'
+
+/**
+ * Create common context-related values from React with a required context.
+ *
+ * @example
+ * ```typescript
+ * const [useFoo, FooConsumer, FooProvider] = createRequiredContext<FooContextValue, 'Foo'>('Foo')
```

Comment:
todo: Add the correct order of values within the tuple which currently do not match what is actually returned from the TS function.